SPECS/xapi-storage.spec

```diff
@@ -103,13 +102,14 @@ cp storage/api/__init__.py storage/api/datapath.py storage/api/volume.py storage
 
 %files ocaml-plugin-devel
 %defattr(-,root,root,-)
+%exclude %{_libdir}/ocaml/xapi-storage/*.ml
 %{_libdir}/ocaml/xapi-storage/*.a
 %{_libdir}/ocaml/xapi-storage/*.cmxa
 %{_libdir}/ocaml/xapi-storage/*.cmx
 %{_libdir}/ocaml/xapi-storage/*.ml
```

Comment:
Yes, I've removed it.   This part is not critical for fixing CA-203817 - the packaging guide says that -devel packages should not usually contain .ml files.